### ISSUE TITLE
some question about the code

### ISSUE THREAD

**Issue by dengismy** [SAI990323/TALLRec#35]
I cannot figure out why does the logits come from the [8241, 3782] in the gengerated scores presenting in the code of evaluate.py, so can someone tell me the reason. I will appreciate it if you can.

**Comment by dengismy** [comment]
i just found it. Someone has given the answer.
![image](https://github.com/SAI990323/TALLRec/assets/53132137/8b60e6ef-7a2b-4e6c-8621-5a17d7c4c03f)
